### ISSUE TITLE
Error in launching simulator in Matlab R2014b

### ISSUE THREAD

**Issue by alekx08** [jdelacroix/simiam#13]
![capture](https://cloud.githubusercontent.com/assets/10465669/5682564/84702054-97e6-11e4-9150-1ffd1165d96d.PNG)

Please help! I am not sure how to proceed so I can start my programming assignments.

**Comment by jdelacroix** [comment]
Hi, the simulator is not yet compatible with the newest Handle Graphics 2 (HG2) system in MATLAB R2014b. You may be able to start MATLAB with the -hg1 option to revert to the older graphics system, which would allow you to run the simulator as-is. The simulator will not work with HG2 until I overhaul the UI.

**Comment by ManuelZ** [comment]
Hi, it just doesn't works for me, I'm trying with "-hg1" and "-hgVersion 1" but I end up with the same error.

**Comment by jdelacroix** [comment]
Yes, that option worked only in previous version to force HG2, but it has been removed in R2014b. I will need to work on porting the simulator to HG2.

**Comment by bausa** [comment]
Any updates on this?

**Comment by rgreen85** [comment]
Any updates on this? It´s a shame this bug renders the software useless for R2014+ users... =(

**Comment by jdelacroix** [comment]
It's in the works, but slow progress for now. I'll post an update when it's ready.

**Comment by rgreen85** [comment]
Thx very much on the prompt reply! Looking forward for it!

**Comment by gmandolesi** [comment]
Hey JP, what's up? Any progress?

**Comment by jdelacroix** [comment]
'hg2' branch in the GitHub repo now hosts the first set of changes to make Sim.I.am work with MATLAB 2014b and HG2. It requires the installation of the GUI Layout Toolbox, which can be downloaded and installed freely from here: http://www.mathworks.com/matlabcentral/fileexchange/47982-gui-layout-toolbox

Not all issues are fixed yet. For example, scrolling by mouse wheel is not working anymore.

**Comment by ManuelZ** [comment]
Hi' I've just seen that the Control of Mobile Robots MOOC is about to open again. But what about this bug, it is finally solved and ready to go into master?

Thank you.

**Comment by jdelacroix** [comment]
I'm not maintaining Sim.I.am for the newest iteration of the course, since I graduated from GT last year. However, I've provided the new maintainer in the GRITS lab with support to get the simulator working with the changes from the 'hg2' branch for all of the assignments. I think it would be useful to keep both 'hg1' and 'hg2' versions on GitHub, because not everyone may be using the latest versions of MATLAB (last course, I got requests for pre R2007b support!).
